index.html

```diff
@@ -49,5 +49,169 @@ <h1 class="h1">I'm Ezechukwu moses Glad to see you!</h1>
         </li>
       </ul>
     </section>
+    <section class="cards">
```

Comment:
- Hey, according to the project requirement, let's use a display property of grid on this container as this project milestone helps test you on using grid and flexbox to layout your page's content. please, let's fix this.

Reply:
Thanks for the review @AbayomiOlaoye